### ISSUE TITLE
v2.3.0 and v.2.4.0 crashes on xamarin forms (android app)

### ISSUE THREAD

**Issue by filoe** [DuendeArchive/IdentityModel.OidcClient#44]
Version 2.3.0 and 2.4.0 crash on xamarin forms (android app).
Seems like the reason is the `System.Runtime.CompilerServices.Unsafe` packages which is a dependency of your packages.
There are several similar issues on the EntityFrameworkCore repository (e.g. https://github.com/aspnet/EntityFrameworkCore/issues/8922).
Also several other related issues like https://forums.xamarin.com/discussion/comment/312241/#Comment_312241.

App starts and crashes with

> D/Mono    ( 3632): Assembly Loader probing location: 'System.Runtime.CompilerServices.Unsafe'.
> F/monodroid-assembly( 3632): Could not load assembly 'System.Runtime.CompilerServices.Unsafe' during startup registration.
> F/monodroid-assembly( 3632): This might be due to an invalid debug installation.

**Comment by leastprivilege** [comment]
All I changed between 2.2 and 2.3 is adding a netstandard2.0 target - the 1.4 is still present - is there a way to force the netstandard version?

**Comment by filoe** [comment]
It comes with the logging stuff which has a dependency which has its own dependencies. 
I've did some dirty trick. I've added 
`<NoWarn>$(NoWarn);NU1605</NoWarn>`
to the csproj file and set the version of the package myself. 
`<PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="4.3.0" />`
Afterwards I've deleted the whole nuget cache on my system, deleted all bin and obj folders in my solution and rebuilt the solution.

It works, but I am not happy with it. I would say, we have to wait until microsoft provides a fix or go back to netstandard 1.4 (hopefully it is still compatible with netstandard 2.0).
